index.html

```diff
@@ -22,6 +22,17 @@
 <head>
     <title>Get HTTPS for free!</title>
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+    <style type="text/css">
+    body {
+	    margin:40px auto;
+	    max-width:650px;
+	    line-height:1.6;
+	    font-size:18px;
+	    color:#444;
+	    padding:0 10px
+    }
+    h1,h2,h3{line-height:1.2}
+    </style>
```

Comment:
Wanted to leave the existing style in tact. The existing ones override the styles I added. Easier to maintain as well.